src/animate-form-elements/styles/scss/animate-form-elements.scss

```diff
@@ -0,0 +1,64 @@
+/* ==========================================================================
+   Animate Form Elements
+   ========================================================================== */
+
+@import "config";
+
+/* Form group
+   ========================================================================== */
+
+.form__input-group {
+    position: relative;
+
+    margin: 3rem 0 0;
+}
+
+/* Inputs
+   ========================================================================== */
+
+.form__input {
+    display: block;
+    padding: .5em;
+
+    background: transparent;
+
+    border: 0;
+    border-bottom: 1px solid rgba($border-color, .3);
+
+    transition: all .3s ease;
+
+    &:active,
+    &:focus,
```

Comment:
Shouldn't there be a light variation on .form__input--active for focused and active state?